index.js

```diff
@@ -27,9 +27,13 @@ const SCHEMA_SLACK_SETTINGS = Joi.object().keys({
         SCHEMA_SLACK_CHANNELS, SCHEMA_SLACK_CHANNEL
     )
 }).unknown(true);
+const SCHEMA_SCM_REPO = Joi.object()
+     .keys({
```

Comment:
indentation is a bit off. https://cd.screwdriver.cd/pipelines/427/builds/74985

Reply:
oops